cluster-autoscaler/FAQ.md

```diff
@@ -217,7 +218,29 @@ More about Pod Priority and Preemption:
  * [Pod Preemption in Kubernetes](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/scheduling/pod-preemption.md),
  * [Pod Priority and Preemption tutorial](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/).
 
+### What namespace should Cluster Autoscaler be deployed into?
 
+Usually, Cluster Autoscaler is expected to be deployed into `kube-system` namespace, which is inline with its semantic purpose -- being a system-level component. However, it can be deployed into a different namespace, with one limitation: Cluster Autoscaler `Role` and `RoleBinding` should remain in `kube-system` namespace:
```

Comment:
Running CA in other namespace on a user node is risky. If CA erases the node it is sitting on and some race condition occurs you may end up with cluster without both CA an capacity to run anything.  Thus it is really important that:

* CA stays in kube-system OR has restrictive PodDisruptionBudget
* CA is run at high priority or is marked as a critical pod.